package.json

```diff
@@ -13,7 +13,8 @@
   "types": "./dist/index.d.ts",
   "scripts": {
     "build": "tsc && rollup --config",
-    "test": "jest --silent=false"
+    "test": "jest --silent=false",
+    "analyze": "depcruise src --include-only \".*(types|index|consts).ts\" --output-type dot | dot -Gsplines=ortho -T svg > dependency-graph.svg"
```

Comment:
I set to visualize public files only, which hides internal file dependency.
This reduces time consumption drastically, and help to inspect user's problem more easily. 

`-Gsplines=ortho` options makes edges in orthogonal manner

Note that this command uses graphviz `dot`. You may have to install it manually, depends on your environment.
For more information, checkout https://www.graphviz.org/download/